x-pack/plugins/lens/public/datasources/form_based/operations/definitions/helpers.tsx

```diff
@@ -138,11 +138,17 @@ export function getSafeName(name: string, indexPattern: IndexPattern | undefined
       });
 }
 
+function areDecimalsValid(inputValue: string | number, digits: number) {
+  const [, decimals = ''] = `${inputValue}`.split('.');
+  return decimals.length <= digits;
+}
+
 export function isValidNumber(
   inputValue: string | number | null | undefined,
   integer?: boolean,
   upperBound?: number,
-  lowerBound?: number
+  lowerBound?: number,
+  digits: number = 2
```

Comment:
why are we limiting it to 2 decimals? I could imagine an SRE wanting up to 4.

Reply:
I see what you mean. It's a simple constant change, so I'll update.

> Yes I agree with that comment, do we need a limit?
Unfortunately yes. I've asked to add the support to `any` but it is not coming in Range components: https://github.com/elastic/eui/issues/5703

Reply:
Done.